modules/n1ql/pages/n1ql-language-reference/using-ai.adoc

```diff
@@ -133,67 +130,120 @@ __optional__
 The Query Service appends a `USE INDEX (USING FTS)` clause to all `FROM` keyspaces in the generated query.
 This enables the query to use a flex index if it's available. 
 --
 
-The Query Service returns an error if you specify a value not included in this list. 
+The statement returns an error if you specify a value not included in this list. 
 
 *Default:* `sql`
 | String
 
 |===
 
 TIP: You can prefix a USING AI statement with EXPLAIN or ADVISE to get the query plan or index recommendations for your generated query.
-Refer to <<example-9>>.
+See <<example-9>>.
+
+[[usage]]
+== Usage
+
+To use a USING AI statement, you must provide your Capella credentials, Capella organization ID, and one or more keyspaces. 
+You can provide these details in two ways: 
+
+* <<request-level-parameters, As request-level parameters>> 
+* <<inline-parameters, Inline within the statement>> 
+
+[[request-level-parameters]]
+=== Set Parameters at the Request Level
+
+You can set `natural_cred`, `natural_orgid`, and `natural_context` as request-level parameters, outside the USING AI statement.
+When set, these parameters apply to all subsequent USING AI statements in that session.
+
+For example:
+
+* In the cbq shell, you can set these parameters using the `\set` command. 
+See <<example-1>>.
+* In the Query Workbench, you can set these parameters in the xref:tools:query-workbench.adoc#query-preferences[Run-Time Preferences] window as named parameters, without the $ prefix. 
+
+For more information about how to set request-level parameters, see xref:n1ql:n1ql-manage/query-settings.adoc#section_nnj_sjk_k1b[Configure Queries].
+
+Once configured, you do not need to specify `creds`, `orgId`, or `keyspaces` in the WITH clause.
+
+[[inline-parameters]]
+=== Set Parameters Inline
+
+You can set the `creds`, `orgId`, and `keyspaces` options directly in the `WITH` clause of the USING AI statement. 
+If specified, these values override the `natural_cred`, `natural_orgid`, and `natural_context` parameters. 
+See <<example-3>>.
 
 == Result
 
 A JSON object containing the generated statement, execution status, and metrics.
 
 If the generated statement is a SELECT query and the `execute` option is `TRUE` (default), the output includes the query results as well.
 
+[[handling-passwords]]
 == Handling Passwords Securely
 
 Use `creds` and `natural_cred` parameters with caution to avoid exposing password information through history files or logs.
 When working with the cbq shell, avoid passing passwords directly on the command line. 
 Instead, use the `\set` command to specify only the username, and then enter the password at the terminal prompt.
 This ensures that the password is not recorded in the shell history.
 
-Similarly, if you are sending requests directly to a REST endpoint from a shell, be mindful of how you provide the password and whether it will be recorded in the shell history.
+For example: 
+
+[source, sh]
```

Comment:
I don't like specifying this as a source block because you get the Copy button, which implies you can just copy the example and paste it into your terminal. Which won't work, because this example is commands and responses: the whole conversation. So just format it as a listing block

```suggestion
```